introspect_query_columns.py

```
@@ -0,0 +1,46 @@
+
+    conn = psycopg2.connect(
+        host=args.host,
+        port=args.port,
+        user=args.user,
+        password=args.password,
+        dbname=args.dbname,
+    )
+    cur = conn.cursor()
+    cur.execute(args.query)
+
+    type_oids = tuple({col.type_code for col in cur.description})
```

Ruff (0.13.1)

Local variable `type_oids` is assigned to but never used

Remove assignment to unused variable `type_oids`

(F841)